server/models/event.js

```diff
@@ -0,0 +1,32 @@
+var mongoose = require('mongoose');
+var Schema = mongoose.Schema;
+
+var categoryValidator = function (categoryArray) {
+  console.log('Category is: ' + categoryArray);
+  console.log(categoryArray.length);
+  if (categoryArray == null || categoryArray.length == 0) {
+    return false;
+  }
+  return true;
+};
+
+var eventSchema = new Schema({
+  eventName: { type: String, required: true },
+  categories: {
+    type: Array,
+    required: true,
+    validate: {
+      validator: categoryValidator,
+      message: 'Categories cannot be empty.'
+    }
+  },
+  numPeople: { type: Number, required: true }, // Do we want ranges? Querying may be better if we have just a number
+  location: { type: String }, // required: true} How are we going to store the location?
+  startTime: { type: Number }, // How are we going to store the time and what time zone will we use as reference?
```

Comment:
Store the time as is for simplicity. The only case I can think of that time is an issue is when people travel out of state into another time zone wanting to pick up some activity.